.travis.yml

```diff
@@ -19,7 +19,9 @@ matrix:
         - php: hhvm
 
 before_install:
+    - if [ "$TRAVIS_PHP_VERSION" != "hhvm" ]; then phpenv config-rm xdebug.ini; fi;
     - composer self-update
+    - if [ "$GITHUB_OAUTH_TOKEN" ]; then composer config --global github-oauth.github.com ${GITHUB_OAUTH_TOKEN}; fi;
```

Comment:
Just realized that this is very important, because otherwise nothing gets cached. :smile: